core/modules/file/file.module

```diff
@@ -461,7 +461,7 @@ function file_entity_info() {
     );
   }
 
-  return array(
+  $return = array(
```

Comment:
Can we give a more meaningful name to this variable?

Reply:
We can. I was just trying to keep it consistent with all of the other implementations of hook_entity_info in core.

Reply:
@klonos Do you think I should change that for all 5 core entities, or just the new instances from this PR?

Reply:
I would prefer all of them please. We need to keep in mind that DX should be improved for novice devs too; and having meaningful, self-explanatory names for variables helps to that end.

Reply:
To me, this is an info hook, so all it does is return info. It seems appropriate that the variable holding the return be called `$return` I think.

Reply:
To me, what is returned here (as the name of the function implies) is information; so I'd expect the variable to be `$info`. The keyword `return` covers the "return" bit.

As a novice developer, it'd make more sense to me to see `return $info;` or `return $entity_info`, rather than `return $return`.

My 2c